### DISCUSSION TITLE
4주차 진행상황 - 남은 이슈 분석

### DISCUSSION THREAD

**Discussion by Miensoap** [boostcampwm-2024/web09-DailyRoad#179] [Show and tell]
### 당장 진행할 수 있는 이슈
~~#178 지도/코스에 추가된 장소 한줄평 (+색) 수정 기능을 구현한다
#177 지도, 코스에서 썸네일 수정 api를 구현한다
#170 트랜잭션이 필요한 부분에 트랜잭션을 적용한다~~ 
**#166 장소가 없는 지도/코스는 표시하지 않는다** -> #168 머지 후 진행
~~#159 사용자는 로그아웃을 할 수 있다~~

### 논의/시간이 필요한 이슈
#172 실제 데이터 확보를 위한 구글 Maps 크롤링을 한다
#171 줌 레벨에 따른 지도 클러스터링 기능을 구현한다 

---

#91 유저/Auth 관련 테스트 코드를 추가한다
#94 무중단 배포를 학습한다
#95 Admin 페이지를 구현한다
#96 ADMIN 페이지 로그인 API 구현한다

### 진행하지 않을 이슈
#161 - 댓글로 사유 남김

**Comment by Miensoap** [comment]
### 5주차 이후 계획

**AI 활용한 기능 추가 (택1) (~5주차)**
  - AI로 검색하기
  - 지도/코스에 추가할 장소 추천받기

**데이터 크롤링 파이프라인 구현 (~5주차)**

**경량화 버전 작성 (~6주차)**
  - 로그 모니터링 등 경량화
  - `Mysql` , `ElasticSearch` 포함한 `Docker-compose` 로 하나의 인스턴스에서 실행할 수 있는 버전
    -  `Nginx` 통합 검토 (설정 파일 미리 작성 등)
  -  구글 검색 기능 모킹이나 비활성화 등 유지 비용을 최소화할 수 있는 버전

**무중단 배포 적용 (~6주차)**

**어드민 페이지 구현 (~6주차)**

**Reply by koomchang** [reply]
무중단 배포 + 오토 스케일링 + 로드밸런싱?

**Comment by koomchang** [comment]
장소 검색 키워드 자동완성 API 도 해봐도 좋을 것 같아요! 프론트에서도 금방처리할 수 있을 것 같아서..
